docs/uptime/settings.asciidoc

```diff
@@ -25,3 +27,20 @@ to manually query Elasticsearch for data outside this pattern!
 See the <<uptime-security,Uptime Elasticsearch Security>>
 and {heartbeat-ref}/securing-heartbeat.html[Heartbeat security]
 docs for more information.
+
+=== Certificate thresholds
+
+The settings page also allows you to specify thresholds for all the TLS certificates monitored
```

Comment:
Active voice: "Specify thresholds for all TLS certificates on the settings page"